jwst/spectral_leak/spectral_leak.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import logging
+from stdatamodels.jwst import datamodels
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def do_correction(sp_leak_ref, ch1b, ch3a):
+    """
+    Short Summary
+    -------------
+    Using the spectral leak reference correction and spectrum containing CH1 B
+    correct the CH 3 A spectrum.
+
+    Parameters
+    ----------
+    sp_leak_ref: ~datamodels.MirMrsPtCorrModel
```

Comment:
Need whitespace before the ":" in all parameter entries

Reply:
Wait, I see `sp_leak_ref` being used as an argument to a datamodels open call down below, so is this already a datamodel when it's passed in as an argument or is it just a file name? If it's already a model, why is it being loaded into a model again down below?